pages/api/webhooks/posts.ts

```diff
@@ -28,14 +29,19 @@ export default async function handler(
     return res.status(401).json({ error: 'Unauthorized' });
   }
 
-  if (!req.body.record) {
+  if (!req.body.record && !req.body.old_record) {
```

Comment:
If you require both, it should be 

```suggestion
  if (!req.body.record || !req.body.old_record) {
```